test/configs/zombieTemplateUpgradeSimple.json

```diff
@@ -35,10 +37,6 @@
             {
                 "name": "charlie",
                 "validator": true
```

Comment:
Why did we remove 4th node?

Reply:
This is my bad, I will fix it

Reply:
We have only dancebox + one container, so I think we can reduce it to 2. Do you want me to try this?

Reply:
Generally, I am in favor of reducing complexity of test setup. So, yeah we can reduce it to 2. But, I would suggest to do it in another PR as this PR is already too big. :)